source/core/src/main/com/csse3200/game/components/items/ItemComponent.java

```diff
@@ -30,18 +30,6 @@ public ItemComponent(String itemName, ItemType itemType, String texturePath) {
 		this(itemName, itemType, "", 0, false, false, texturePath);
 	}
 
-	/**
-	 * Constructor for Item
-	 *
-	 * @param itemName user facing name for item
-	 * @param itemType the enum for type of item
-	 * @param perishable whether the item is perishable
-	 * @param texturePath path for the item texture
-	 */
-	public ItemComponent(String itemName, ItemType itemType, String texturePath, boolean perishable) {
-		this(itemName, itemType, "", 0, false, perishable, texturePath);
-	}
-
```

Comment:
Why is this removed?

Reply:
I removed all references to perishable items because its not being used .